modules/RTC/TraceablePeerConnection.js

```diff
@@ -328,12 +326,28 @@ export default function TraceablePeerConnection(
     this._localTrackTransceiverMids = new Map();
 
     /**
-     * Holds the SSRC map for the local tracks.
+     * Holds the SSRC map for the local tracks mapped by their source names.
      *
-     * @type {Map<string, TPCSSRCInfo>}
+     * @type {Map<string, TPCSourceInfo>}
+     * @property {string} msid - The track's MSID.
+     * @property {Array<string>} ssrcs - The SSRCs associated with the track.
+     * @property {Array<TPCGroupInfo>} groups - The SSRC groups associated with the track.
+     * @property {string} msid - The track's MSID.
      */
     this._localSsrcMap = null;
 
+    /**
+     * Holds the SSRC map for the remote tracks mapped by their source names.
+     *
+     * @type {Map<string, TPCSourceInfo>}
+     * @property {string} mediaType - The media type of the track.
+     * @property {string} msid - The track's MSID.
+     * @property {Array<string>} ssrcs - The SSRCs associated with the track.
+     * @property {Array<TPCGroupInfo>} groups - The SSRC groups associated with the track.
+     * @property {string} msid - The track's MSID.
+     */
+    this._remoteSsrcMap = null;
```

Comment:
Looking at how we need to handle the code later on, doesn't it make sense to always have this initialized to a an empty Map?

Having its size be 0 would be equivalente to the current null value, but I bet it sill simplify some code and make things easier to work with.

Reply:
I have a follow up PR that will get rid of localSSRCs/remoteSSRCs and start using _localSSCMap and _remoteSSRCMap for all other helper functions, I have them initialized to an empty map there. I should have done that here itself :) Modified it.